prep_data/image_data_guide/03b_tensorflow_preprocessing.ipynb

```diff
@@ -68,7 +68,7 @@
     "<a id='ipg3b.1'></a>\n",
```

Comment:
<p>I think <code>tfds.ImageFolder</code>  cannot automatically create a dir for you, so you will need to create <code>./data_structured</code>  yourself. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2083/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>